deploy/helm/sumologic/conf/logs/logs.source.containers.conf

```diff
@@ -25,7 +25,7 @@
   #  only match fluentd logs based on fluentd container name. by default, this is
   #  <filter **collection-sumologic**>
 {{ if .Values.fluentd.logLevelFilter }}
-  {{ printf "<filter **%s**>" (include "sumologic.fullname" .) }}
+  {{ printf "<filter **%s**>" (include "sumologic.metadata.name.fluentd" .) }}
```

Comment:
Where does it come from @sumo-drosiek ?

Reply:
It's from newly created https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/960/files#diff-2794b50c6e48ee4f11c58c42158629b0R196

Reply:
Ahh, somehow I must have missed it when looked at changes, thank you!